prep-css-flexbox/align-items.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      flex-basis: calc(100% / 3);
```

Comment:
Hi Chase for each of your HTML documents, be sure to apply the `flex-basis` property to the rule set for `.card-wrapper` instead of the `#card-deck`.